resources/i18n/fr_FR/cura.po

```diff
@@ -5291,3 +5291,86 @@ msgstr "Générateur de G-Code compressé"
 msgctxt "description"
 msgid "Provides support for exporting Cura profiles."
 msgstr "Fournit la prise en charge de l'exportation de profils Cura."
+
+msgctxt "@info:title"
+msgid "Error"
+msgstr "Erreur"
+
+msgctxt "@action:button"
+msgid "Cancel"
+msgstr "Annuler"
+
+msgctxt "@title:tab"
+msgid "Settings"
+msgstr "Paramètres"
+
+#, python-brace-format
+msgctxt "@label Don't translate the XML tag <filename>!"
+msgid "The file <filename>{0}</filename> already exists. Are you sure you want to overwrite it?"
+msgstr "Le fichier <filename>{0}</filename> existe déjà. Êtes-vous sûr de vouloir le remplacer?"
+
+msgctxt "@action:button"
+msgid "Learn more"
+msgstr "En savoir plus"
+
+msgctxt "@title:tab"
+msgid "General"
+msgstr "Général"
+
+msgctxt "@label (%1 is object name)"
+msgid "Are you sure you wish to remove %1? This cannot be undone!"
+msgstr "Êtes-vous sûr de vouloir supprimer l'objet %1 ? Vous ne pourrez pas revenir en arrière !"
+
+msgctxt "@label"
+msgid "Type"
+msgstr ""
```

Comment:
I really didn't expect anything in French to be untranslated ;-)

Reply:
In this case, the translation is "Type" so it would have been unnoticed :laughing: